### PR TITLE
Chloe cell

### PR DESCRIPTION
**Pull request by chloell5**
Completed render method functionality
Finished work on render method tests

### CODE REVIEW

File: spec/cell_spec.rb [hunk 1]
```diff
@@ -81,3 +81,59 @@
...
+
+      expect(cell_2.render(true)).to eq('S')
+    end
+
+    it 'renders as H if ship is hit' do
+      cell_2 = Cell.new('C3')
+      cruiser = Ship.new('Cruiser', 3)
+      cell_2.place_ship(cruiser)
+      cell_2.fire_upon
+      cruiser.sunk?
```
Comment (hschmid516) [inline spec/cell_spec.rb]: We should change `cruiser.sunk? ` to a test here to show that it returns false, and in the next test to show that it returns true. If the interaction pattern shows the return, I think we should replicate it in the tests.

**Review by hschmid516 — COMMENTED** [review]
Just one minor comment before we merge, but it looks great overall!

**Comment by chloell5** [comment]
That should take care of that!

File: spec/cell_spec.rb (hunk 1 above)
Comment (chloell5) [reply spec/cell_spec.rb]: Fixed, I think

Comment (hschmid516) [reply spec/cell_spec.rb]: You can remove `cruiser.sunk?` in lines 122 and 135 because they aren't doing anything.

Comment (chloell5) [reply spec/cell_spec.rb]: Should be all clear now

**Comment by hschmid516** [comment]
Wooo iteration 1 done!

**Pull request merged**